ONSdigital/eq-survey-runner
# Use schema_name and allow census claims

Context available in [trello](https://trello.com/c/ZhMmtMYB/2954-use-schemaname-instead-of-eqid-formtype)

Removes the use of `eq_id` and `form_type`. These are now equivalent to `schema_name`
Runner uses `schema_name` throughout. The census claims:
`survey` / `case_type` / `region_code` can now be used instead of `schema_name`

Additionally, the schema endpoint has been updated to only send filenames without
the file extension. This now matches the format of `schema_name`. The file extension seems like an internal implementation detail.

Basic validation for the census parameters has been added using marshmallow. Marshmallow will also insert a `schema_name` parameter into the metadata as soon as it has been validated.

`instrument_id` in the downstream payload has been changed to `schema_name`.

### How to review 
- This should be tested in combination with the changes to eq-questionnaire-launcher.

- Ensure that runner works when questionnaire-launcher passes just a schema_name (on a test schema)
- Ensure runner works when questionnaire-launcher passes the survey / case_type / region_code

### Checklist
